### DISCUSSION TITLE
Trying to add Fooocus-API to StabilityMatrix

### DISCUSSION THREAD

**Discussion by yak1ex** [mrhan1993/Fooocus-API#359] [Ideas]
Thank you very much for developing Fooocus-API.
By using this, I can avoid huge struggles with Gradio API.

Now, I'm trying to add Fooocus-API as a [StabilityMatrix](https://github.com/LykosAI/StabilityMatrix) package at https://github.com/yak1ex/StabilityMatrix/tree/add_a_package_for_foocus-api
Currently, at least, it is possible to install and run Fooocus-API via StabilityMatrix.
I'm using StabilityMatrix to install Fooocus, and Fooocus-API can be shared with many part of Fooocus, so it can get a great benefit from model sharing by StabilityMatrix.

Though I'm not sure if a PR is accepted by StabilityMatrix devs, I plan to try to make the PR anyway.
Are there any issues for you?

If it is acceptable, there is one more question.
StabilityMatrix has a banner image for each package. I currently use the GitHub repository banner as a placeholder
<img src="https://opengraph.githubassets.com/9ee19e1641a0080ef654432d5ad6c24e2cf6a8c4f76c54fa14712dce8772c0be/mrhan1993/Fooocus-API" width="300">
Is there any preference for the banner image?

**Comment by yak1ex** [comment]
@mrhan1993 I'm concerned that there is no notification to you. So, I added this mention. If this has already been brought to your attention, please disregard this comment.

**Comment by yak1ex** [comment]
I created a banner image that combines API documentation from FastAPI, an example request code, its response JSON and an image.
![Fooocus-API](https://github.com/mrhan1993/Fooocus-API/assets/820530/952c9777-8d57-4b7e-8bd3-f574d508ebee)

If there are no objections, I plan to submit a pull request to StablityMatrix next week, that includes this image.

**Comment by yak1ex** [comment]
Though I have noticed the new project https://github.com/mrhan1993/FooocusAPI, I make a pull request anyway: 
https://github.com/LykosAI/StabilityMatrix/pull/753

**Comment by mrhan1993** [comment]
I personally will not place any restrictions on the use of this project. You can do whatever you want.
